<ui version="4.0" >
 <class>LicenseInstallerWidgetBase</class>
 <widget class="QDialog" name="LicenseInstallerWidgetBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>License Installer</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>&amp;CS8C controller IP Address:</string>
       </property>
       <property name="buddy" >
        <cstring>leAddress</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="leAddress" >
       <property name="inputMask" >
        <string>009.009.009.009;_</string>
       </property>
       <property name="text" >
        <string>192.168.000.005</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>&amp;Machine Number:</string>
       </property>
       <property name="buddy" >
        <cstring>leMachineNumber</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="leMachineNumber" >
       <property name="inputMask" >
        <string>F99_NNNNNN_A_99; </string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>&amp;License Key:</string>
       </property>
       <property name="buddy" >
        <cstring>leKey</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="leKey" />
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btInstall" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>&amp;Install</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btClose" >
       <property name="text" >
        <string>Cl&amp;ose</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btClose</sender>
   <signal>clicked()</signal>
   <receiver>LicenseInstallerWidgetBase</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>302</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel" >
     <x>305</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
